Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs

```diff
@@ -439,14 +438,8 @@ public string StatusWindowURL
         {
             get 
             { 
-                try

```

Comment:
If you remove the try/catch here, the trayicon could crash if `GetAuthToken()` fails?